### DISCUSSION TITLE
ScriptReference/SceneManagement.EditorSceneManager-playModeStartScene

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#662] [Notes]
# ScriptReference/SceneManagement.EditorSceneManager-playModeStartScene

https://docs.unity3d.com/ScriptReference/SceneManagement.EditorSceneManager-playModeStartScene.html

**Comment by andreasIsCoding** [comment]
Note that this field persists across domain reloads, so you'll need to manually set it to null to restore default behavior.
